pkg/test/framework/components/apps/kube.go

```diff
@@ -664,19 +666,6 @@ func (a *kubeApp) Call(e AppEndpoint, opts AppCallOptions) ([]*echo.ParsedRespon
 		return nil, err
 	}
 
-	if len(resp) != 1 {
```

Comment:
I think we should leave the check for len here.  Everything else we can remove completely.